README.md

```diff
@@ -36,74 +36,71 @@ The Halkyon Operator requires `Kubernetes >= 1.13` or `OpenShift >= 3.11`.
 
 ## Key concepts
 
-A modern application, which is defined as a collection of micro-services as depicted hereafter
+An example of a simple, modern application, defined as a collection of micro-services is depicted hereafter
 
 ![Composition](component-operator-demo.png)
 
-will require several Kubernetes resources in order to be deployed on a kubernetes cluster. Furthermore several development iterations are usually required to make the 
-application production ready.
+Such an application, though simple, will require several Kubernetes resources in order to be deployed on a Kubernetes cluster. 
+Furthermore several development iterations are usually required to make the application production ready.
 
-When, during the analysis phase, the developers will discuss the final picture about the solution to be designed, they
-will certainly adopt this very straightforward convention (aka Fluent DSL) to express the different micro-services, their relations and ultimately
-the services needed.
+The entry point to the application is the `fruit-client-sb` application. It connects to a backend REST endpoint implemented
+by the `fruit-backend-sb` application. This application, in turns, uses the services of a PostgreSQL database.
 
-`(from:componentA).(to:componentB).(to:serviceA)`
```

Comment:
I disagree that you remove the DSL language like also the text about how developers, analysts will use it to describe, talk about the application to be developed

Reply:
I personally find it confusing so I agree with Chris on removing it

Reply:
I don't see the point of introducing a DSL that's not used anywhere else at this point. If we made use of it, why not but that's not the case and doesn't really add anything to the point we're trying to make, in my opinion.

Reply:
ok. I will then accept this PR and add later a footnote about this DSL language